### ISSUE TITLE
CSIS study - coastal cities area selection

### ISSUE THREAD

**Issue by mattia-leone** [clarity-h2020/csis#123]
Selecting a box with a portion in the sea creates this issue

<img width="1275" alt="Schermata 2019-11-26 alle 10 52 13" src="https://user-images.githubusercontent.com/56071027/69619293-abd1f500-103b-11ea-8cc3-c812f280aa4d.png">

The portion of sea should be included in the city borders to perform the study

**Comment by DenoBeno** [comment]
An alternative would be to allow users to choose an area as long as *part* of it is within a city area. Not sure if the calculations woudl perform properly in this case -- @humerh , WDYT?

**Comment by p-a-s-c-a-l** [comment]
@therter Is that error message coming from the map component? Why is it even possible, to select a study area outside of the city polygon?

**Comment by humerh** [comment]
From my side I see no problem.

**Comment by therter** [comment]
I can import the envelope of the city polygon instead of the detailed polygon. This should solve the problem. Should I do this?

**Comment by p-a-s-c-a-l** [comment]
> This should solve the problem.

I don't think so. What happens if somebody selects a study area in the gulf of Naples? What would the model calculate then? Nothing useful I suppose.

As Denis suggested we should check whether the study area *intersects* with the city polygon. ATM you are checking for "contains", right?
